### PR TITLE
Show stale service IDs on deployments

### PR DESCRIPTION
**Pull request by philipnrmn**
**Background** - when a service is deploying for the first time, rolling back its deployment removes the service configuration. When this happens it's possible for a deployment to exist with references to services which no longer exist in the `MarathonStore`/`DCOSStore`. 

**Previous behaviour** - the deployment was shown but the deleted services were not.

**New behaviour** - the deployment is show and the deleted service IDs are also shown.

Paired with @mlunoe

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

**Comment by orlandohohmeier** [comment]
The code looks good to me! But I've difficulties to test this. Could you tell me what I need to do to "create" a rather long rollback deployment?

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
Tests are failing. `12 tests failed, 1853 tests passed (1865 total in 179 test suites, run time 159.699s)`

File: src/js/pages/services/DeploymentsTab.js [hunk 1]
```diff
@@ -95,17 +96,31 @@ class DeploymentsTab extends mixin(StoreMixin) {
...
         {this.renderAffectedServicesList(deployment.getAffectedServices())}
+        {this.renderStaleServicesList(deployment.getStaleServiceIds())}
       </dl>
     );
   }
 
+  renderStaleServicesList(serviceIds) {
+    return serviceIds.map(function (serviceId, index) {
+      return (
+        <dd key={`stale_${index}`}>
```
Comment (kennyt) [inline src/js/pages/services/DeploymentsTab.js]: is it possible for us to have the key as `stale_${serviceId}` rather than relying on the index?

**Comment by philipnrmn** [comment]
Tests are now fixed, apologies for that. 

@orlandohohmeier I ended up manipulating the API data to get a long rollback - they appear to be almost instantaneous in every case I tried otherwise.

File: src/js/pages/services/DeploymentsTab.js [hunk 2]
```diff
@@ -95,17 +96,34 @@ class DeploymentsTab extends mixin(StoreMixin) {
...
+        {this.renderStaleServicesList(deployment.getStaleServiceIds())}
       </dl>
     );
   }
 
+  renderStaleServicesList(serviceIds) {
+    if (serviceIds == null) {
+      return;
+    }
+    return serviceIds.map(function (serviceId, index) {
```
Comment (Poltergeist) [inline src/js/pages/services/DeploymentsTab.js]: `index` is defined but never used :-(

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
It looks good! 👍  🚢 🍧

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Pull request merged**